gulpfile.js

```diff
@@ -19,10 +19,13 @@ function getFolders(dir) {
         });
 }
 
-var transifex = require('gulp-transifex').createClient(options);
+const options         = require('./transifex-config.json');
+const transifex       = require('gulp-transifex').createClient(options);
+const languagesFolder = './languages/';
+const folders         = getFolders(languagesFolder);
 
 // Create POT out of PHP files
-gulp.task('prepare-source', function () {
+function prepare_source() {
     gulp.src('**/*.php')
```

Comment:
In new gulp we are supposed to return a promise or promise-like to properly signal a task completion. Please read documentation here https://gulpjs.com/docs/en/getting-started/async-completion

So, for all gulp task-related functions, you need to return the stream. 

```js
function foo() {
  return gulp.src(...).pipe(...);
}
```

Then gulp will correctly hint to us about the time it took.

When there are no streams to return, you have various options. I find using the built-in Promise to be very powerful and flexible.

```js
return new Promise((resolve, reject) => {
  // doSomething is an async task, but it is implemented using a callback pattern.
  doSomething(() => {
    resolve();
  });
});
```

You can also use the `reject` to tell gulp the task has failed.